vitrina/datasets/models.py

```diff
@@ -272,7 +289,9 @@ class Dataset(TranslatableModel):
         autocomplete_view="autocomplete_tags",
         autocomplete_limit=20,
         verbose_name=_("Žymės"),
-        help_text=_("Pateikite kableliu atskirtą sąrašą žymių."),
+        help_text=_(
+            "Duomenų išteklių apibūdinantis raktažodžiai arba žymos. Atitinka dcat:keyword."
+        ),
```

Comment:
```suggestion
        help_text=_(
            "Duomenų išteklių apibūdinantys raktažodžiai arba žymos. Atitinka dcat:keyword."
        ),
```

Also do not know if `Duomenų išteklių` is singular, maybe `Duomenų išteklį`? Although both sound wrong..

Reply:
Yeah, I had this same thought. "Duomenų išteklių" is already used in other place so I used it here too